yt/visualization/volume_rendering/transfer_functions.py

```diff
@@ -686,7 +694,7 @@ def y_format(y, pos):
         ax.set_xlim(0.0, max_alpha)
         ax.get_xaxis().set_ticks([])
         ax.set_ylim(visible[0].item(), visible[-1].item())
-        ax.tick_params(axis="y", colors="white", size=10)
+        ax.tick_params(axis="y", colors="white", labelsize=label_fontsize)
```

Comment:
Took me long enough (sorry again) but I just realised that you're exposing the *tick* label size, not the size of the colorbar label itself (in fact, it's already exposed as the `size` parameter). I think the parameter would be better named `tick_label_size`.

I also see now that the actual colorbar label size is automatically scaled according to the resolution, so wouldn't it make sense to do the same thing with tick labels so their size, relative to the colorbar label, would be resolution-independent ?
There's still value in exposing both parameters independently as you're doing here, so we can keep the bulk of your patch as is, but we'd need remove references to special strings from matplotlib.

Reply:
all of the label parameters (this would make 3) are for the tick labels.  So if you change this one, all 3 should be changed.

Reply:
ah, I see. Let's keep the name then. What about auto-scaling ?